### PR TITLE
Dario

### PR DESCRIPTION
**Pull request by dariomolina93**

### CODE REVIEW

File: auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java [hunk 1]
```diff
@@ -46,14 +50,8 @@ public void onClick(View v){
...
-                String sms = "This is going to be a really long message and it will take some time to get there,  This is going to be a really long message and it will take some time to get there";
-
-                SMS message = new SMS(getApplicationContext());
-                message.sendSMS("8317760235",  sms);
-
-
-
-
+                String sms = "this is going to be around 160 chars. testing. this is going to be around 160 chars. testing.this is going to be around 160 chars. testing.this is going to be .";
+                message.sendSMS("4159881453",  sms);
```
Comment (lloydchang) [inline auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java]: @dariomolina93 curiously, who's phone number is 4159881453 ?  Thanks.

File: auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java [hunk 2]
```diff
@@ -37,12 +43,34 @@ public void onClick(View v) {
...
                 AffidivitApplication application= DataUtils.getAffidivitApplication().build();
-                new TwilioAPI().execute(application);
-                Toast.makeText(MainLobbyActivity.this, "Parientes notificados", Toast.LENGTH_SHORT).show();
+//                new TwilioAPI().execute(application);
+//                Toast.makeText(MainLobbyActivity.this, "Parientes notificados", Toast.LENGTH_SHORT).show();
+
+                String sms = "This is going to be a really long message and it will take some time to get there,  This is going to be a really long message and it will take some time to get there";
+
+                SMS message = new SMS(getApplicationContext());
+                message.sendSMS("8317760235",  sms);
```
Comment (lloydchang) [inline auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java]: @dariomolina93 curiously, who's phone number is 8317760235 ? Thanks.

**Comment by lloydchang** [comment]
Thank you @dariomolina93 for the pull request — I added my comments to this review.  As you requested, I asked @J0S349 and @bsanchezg / @benitosanchez to review when their time kindly permits :-)

File: auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java (hunk 1 above)
Comment (dariomolina93) [reply auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java]: @lloydchang It's actually my phone number, I forgot to remove it lol. I'm going to cancel this PR and submit another with cleaner code. Thanks!

Comment (lloydchang) [reply auxilio/app/src/main/java/com/auxilio/MainLobbyActivity.java]: Thank you @dariomolina93; sounds good :-)

**Pull request closed without merging**